.github/workflows/enso4igv.yml

```diff
@@ -0,0 +1,41 @@
+# This workflow will build a Java project with Maven, and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Enso Language Support for IGV
+
+on:
+  push:
+    branches: [develop]
+  pull_request:
+    branches: [develop]
+  paths:
+    - "tools/enso4igv/**/*"
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        java: ["8"]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Java
+        uses: actions/setup-java@v2
+        with:
+          distribution: "zulu"
+          java-version: ${{ matrix.java }}
+          cache: maven
+
+      - name: Update project version
+        run: |
+          cd tools/enso4igv
+          mvn versions:set -DnewVersion=`mvn -q -DforceStdout help:evaluate -Dexpression=project.version | cut -f1 -d -`.$GITHUB_RUN_NUMBER
```

Comment:
Trickish, at least for a Windows user. :D 
If it has to be, please comment.

Reply:
> Trickish, ... please comment.

I want to invoke
```
mvn -q -DforceStdout help:evaluate -Dexpression=project.version
```
which outputs something like `1.0-SNAPSHOT`. The next `cut` command remove the suffix, so we get `1.0` and then we append `$GITHUB_RUN_NUMBER` to form something like `1.0.34`. The resulting _semantic version_ is then taken are applied to the `pom.xml` by running:
```
mvn versions:set -DnewVersion=1.0.34
```
Ideas how to rewrite the _oneliner_ into proper GitHub Actions style welcomed.